Sources/Responses/ZipDownload.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ZipDownload.swift
+//  BoxSDK-iOS
+//
+//  Created by Skye Free on 1/11/21.
+//  Copyright © 2021 box. All rights reserved.
+//
+
+import Foundation
+
+/// Defines a Zip Download
+public class ZipDownload: BoxModel {
+    // MARK: - Properties
+
+    public private(set) var rawData: [String: Any]
+
+    /// The URL to download the Zip file. If entered in a browser, this URL will trigger a download of the Zip file.
+    public let downloadUrl: URL
+    /// The URL to monitor the status of the download.
+    public let statusUrl: URL
+    /// Expiration date of the Zip file download.
+    public let expiresAt: Date?
+    /// Conflicts that occur between items that have the same name.
+    public let nameConflicts: [[[String: String]]]?
```

Comment:
Sorry, should have mentioned this in the previous review, but forgot. What do you think about making an object for a single name conflict, so name conflicts is a list of name conflict objects? It is something we did in the java SDK [here](https://github.com/box/box-java-sdk/pull/825/files#diff-5ab88aec08abb5a7ef144a623ccfd0d98a63360e3d290fe2a54e791971adfd79). Do you think it could be useful to make nameConflicts more clear or not really?

Reply:
Ah good point, I think that'll make nameConflicts clearer too. I'll add that in!